apps/server/cypress.config.ts

```diff
@@ -4,6 +4,7 @@ export default defineConfig({
   e2e: {
     retries: 1,
     defaultCommandTimeout: 15000,
+    specPattern: 'cypress/e2e/suite2/**/*.{js,jsx,ts,tsx}',
```

Comment:
You mean suite1 right?

Reply:
Yeah... I don't know how the tests passed with that typo. Something weird is going on.